tests/index.test.tsx

```diff
@@ -2,8 +2,20 @@ import renderer from 'react-test-renderer'
 import { render, screen } from '@testing-library/react'
 import Index from '../pages/index'
 
+import { useRouter } from 'next/router'
+
+jest.mock('next/router', () => ({
+  __esModule: true,
+  useRouter: jest.fn(),
+}))
+
 describe('Exemplar-react-test renderer Snapshot test', () => {
   it('renders homepage unchanged', () => {
+    const mockRouter = {
+      locale: 'en',
+    }
+    ;(useRouter as jest.Mock).mockReturnValue(mockRouter)
```

Comment:
whoa, does this semicolon need to be at the start here because our linter is disallowing them at the end of lines??

Reply:
full honesty this was snagged wholesale from a guide 😅

Reply:
haha ok let's err on the side of commenting anything out of the ordinary (in this case, either remove the ; if not needed, or comment with why it's needed, and include a link to the guide if that's helpful)